week1/utils.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+	"strconv"
+)
+
+func add(x, y float64) float64 {
+	return x + y
+}
+
+func subtract(x, y float64) float64 {
+	return x - y
+}
+
+func multiply(x, y float64) float64 {
+	return x * y
```

Comment:
you're using 2 styles (named return & value return) of returning in 1 file, please pick one and stick to it